src/Sylius/Bundle/ApiBundle/Resources/config/services.xml

```diff
@@ -31,8 +31,12 @@
             <argument type="service" id="api_platform.symfony.iri_converter" />
         </service>
 
+        <service id="Sylius\Bundle\ApiBundle\Controller\UploadProductImageAction" public="true">
```

Comment:
Outside of scope of current PR,
there are plenty of controllers, so I think it's a good time to move definitions to `services/controllers.xml`